vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,148 @@
+# Vendor onboarding checklist
+
+## Purpose
+
+This document supports the Division of Public Assistance Eligibility
+Information System Replacement (EIS-R) Project. Specifically, this
+document addresses our agile acquisition practice and onboarding new
+vendor staff. Within agile acquisition, we anticipate bringing in a
+potentially different vendor every three to four months, possibly more
+often when we begin running multiple acquisitions simultaneously. With
+short timelines, this onboarding checklist will bring new vendor staff
+up to speed as quickly as possible.
+
+# Audience
+
+This document intends to inform EIS-R State of Alaska project staff and
+the vendors we engage with in various product-increment acquisitions.
+
+# Roles
+
+This section identifies the anticipated vendor roles that may be
+engaged with the
+project.  Note that this list is subject to change depending on the approaches that vendors 
+propose for staffing and managing their teams.
+
+| **Role**                       | **Description**                                                                                                                                                |
+| ------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Project manager / Scrum master | Manage the project, promote scrum process and lead daily scrums on vendor team                                                                                 |
+| Technical lead                 | Lead the technical work                                                                                                                                        |
+| Developers                     | Develop client and server application components using CSS, HTML5, Javascript and .NET Core                                                                    |
+| Researcher                     |                                                                                                                                                                |
+| User experience designer       | Design the user experience to achieve good continuity between new and legacy application and usability that yields the intended value of the product increment |
+| Visual designer                |                                                                                                                                                                |
+
+# Tools and resources

```

Comment:
I think that this table should be broken down into 2 different ones. The first a list of Tools/Resources and the purpose. The second table, a matrix of Tool/Resources to Roles.

![tools-resources-purpose](https://user-images.githubusercontent.com/33638291/32753627-15fe0d8a-c87a-11e7-97f2-7d9e7399a0f5.PNG)
![tools-resources-roles](https://user-images.githubusercontent.com/33638291/32753628-1622ad16-c87a-11e7-8b52-31fa147fbf19.PNG)

Reply:
I like that idea, @dvvarela!  I do think that for VSTS and the AK Dev Env line items we'll need to be more specific than "RW" (read/write), but I like the matrix!